genesyscloud/resource_genesyscloud_outbound_contactlist_test.go

```diff
@@ -0,0 +1,269 @@
+					previewModeColumnNameUpdated,
+					previewModeAcceptedValuesUpdated,
+					columnNames,
+					automaticTimeZoneMapping,
+					zipCodeColumnName,
+					"genesyscloud_outbound_attempt_limit."+attemptLimitResourceID+".id",
+					generatePhoneColumnsBlock(
+						"Cell",
+						"cell",
+						"",
+					),
+					generatePhoneColumnsBlock(
+						"Home",
+						"home",
+						"",
+					),
+				),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "name", nameUpdated),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "division_id", "4a7e4f4b-e734-40b6-838e-557c5eedd49a"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "column_names.0", "Cell"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "column_names.1", "Home"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "column_names.2", zipCodeColumnName),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "zip_code_column_name", zipCodeColumnName),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "phone_columns.0.column_name", "Cell"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "phone_columns.0.type", "cell"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "phone_columns.1.column_name", "Home"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "phone_columns.1.type", "home"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "preview_mode_column_name", previewModeColumnNameUpdated),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "preview_mode_accepted_values.0", previewModeColumnName),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "preview_mode_accepted_values.1", previewModeColumnNameUpdated),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "automatic_time_zone_mapping", automaticTimeZoneMapping),
+					resource.TestCheckResourceAttrPair("data.genesyscloud_outbound_attempt_limit."+attemptLimitDataSourceID, "id",
+						"genesyscloud_outbound_contact_list."+resourceId, "attempt_limit_id"),
+				),
+			},
+			{
+				ResourceName:      "genesyscloud_outbound_contact_list." + resourceId,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+		CheckDestroy: testVerifyContactListDestroyed,
+	})
+
+	automaticTimeZoneMapping = falseValue
+	callableTimeColumn1 := "America/Chicago"
+	callableTimeColumn2 := "Ireland/Galway"
+
+	// Test with callable_time_column field set
+	// (Can only be set when 'automatic_time_zone_mapping' is 'false', and 'automatic_time_zone_mapping' cannot be updated)
```

Comment:
If this can't be updated should it be set to ForceNew?

Reply:
@ronanwatkins Done ✅